nominees/coinsence.json

```diff
@@ -1,16 +1,31 @@
 {
   "name": "Coinsence",
-  "description": "Platform to match youth volunteers to social enterprises",
+  "aliases": [
+    ""
+  ],
+  "description": "A global decentral collaboration platform and change-maker network powered by blockchain based impact currencies",
```

Comment:
Is it possible to get a description with less jargon? It is hard to understand what the product is from this line.